internal/storage/db/mysql/schema.sql

```diff
@@ -7,6 +7,7 @@ CREATE TABLE IF NOT EXISTS policy (
     kind VARCHAR(128) NOT NULL,
     name VARCHAR(1024) NOT NULL,
     version VARCHAR(128) NOT NULL,
+    fqn VARCHAR(255) NOT NULL,
```

Comment:
FQN is the combination of `cerbos` + `kind` + `name` + `version`.  So, given that the name column is 1024 chars, FQN should be more than that. 

I think that instead of the FQN, we should let the human readable identifier be the tuple (`kind`, `name`, `version`). That saves the users from being exposed to internal implementation details and work with the concrete data they deal with in the policy files.